api_tests/src/actors/swap_factory.ts

```diff
@@ -23,16 +23,16 @@ import { HbitRequestParams } from "comit-sdk/dist/src/cnd/swaps_payload";
 
 declare var global: HarnessGlobal;
 
-interface Ledgers {
-    alpha: keyof AllWallets;
-    beta: keyof AllWallets;
+interface SwapSettings {
+    ledgers?: { alpha: keyof AllWallets; beta: keyof AllWallets };
+    instantRefund?: boolean;
```

Comment:
:joy:

Reply:
You hurt me, @thomaseizinger :sob:

Reply:
Sorry, I didn't mean to :)
I was just positively amused!